easybuild/easyblocks/g/go.py

```diff
@@ -61,7 +61,7 @@ def install_step(self):
 
         # $GOROOT_FINAL only specifies the location of the final installation, which gets baked into the binaries
         # the installation itself is *not* done by the all.bash script, that needs to be done manually
-        cmd = "GOROOT_FINAL=%s ./all.bash" % self.installdir
+        cmd = "GOROOT_BOOTSTRAP=%s GOROOT_FINAL=%s ./all.bash" % (os.environ.get('EBROOTGO'),self.installdir)
```

Comment:
looks good, but you should use the `get_software_root` function rather than hardcoding `$EBROOTGO`

or, actually, how can `EBROOTGO` be defined in the environment when you're installing Go?!

maybe you mean this?

```python
cmd = "GOROOT_BOOTSTRAP=%(installdir)s GOROOT_FINAL=%(installdir)s ./all.bash" % {'installdir': self.installdir}
```